MicrosoftFluentUI.podspec

```diff
@@ -97,6 +97,7 @@ Pod::Spec.new do |s|
     drawer_ios.dependency 'MicrosoftFluentUI/ResizingHandleView_ios'
     drawer_ios.dependency 'MicrosoftFluentUI/Separator_ios'
     drawer_ios.dependency 'MicrosoftFluentUI/TouchForwardingView_ios'
+    drawer_ios.dependency 'MicrosoftFluentUI/Vnext_ios'
```

Comment:
What else is needed in Core_ios so you can drop the Vnext_ios subspec?

Reply:
`Vnext/Core/**` were required. I have updated the changes to merge in core subspec instead of vnext_ios